main.py

```diff
@@ -346,8 +347,7 @@ async def submit_experiment(  # pylint: disable=too-many-arguments, too-many-loc
           None for no due date.
         visualize: If True, the experiment file is modified for visualization.
           The original file and vcd file are saved in the visualize path set in config.json.
-        cls: The class name of the experiment.
-          It is only necessary when the visualize argument is True.
+          The cls argument should be given.
```

Comment:
Why is this line required?

Reply:
Umm.. It looks wrong. I will remove it.

Reply:
Oh, so you mean the `cls` should be given if `visualize` is `True`, right?

Reply:
Yes! You're right.